pallets/identity/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+//! Runtime API definition for Identity module.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Codec, Decode, Encode};
+use polymesh_primitives::{IdentityId, Ticker};
+
+sp_api::decl_runtime_apis! {
+    pub trait IdentityApi<IdentityId, Ticker> where
+        IdentityId: Codec,
+        Ticker: Codec,
+    {
+        fn is_identity_has_valid_cdd(did: IdentityId, buffer_time: Option<u64>) -> (bool, Option<IdentityId>);
```

Comment:
1. Consider returning a stroger type like Result instead of a tuple. 
2. Also, instead of retuning an option type, using a Result type that encapsutates the optional value with success/failure reasons encodes better.
For e.g., see `FutureResult` type. It need not be `FutureResult` any other new type would also work.